lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
(The problem I am trying to ensure we are solving is that the setup is complete before the mark is created. Two promises in the same scope without waiting is still open to races since they are ensured async.)